features/step_definitions/government_steps.rb

```diff
@@ -20,3 +20,7 @@
     end_date: end_date
   })
 end
+
+Given(/^there is a current government$/) do
+  FactoryGirl.create(:current_government)
```

Comment:
The most common style in whitehall is to simply call `create` instead of `FactoryGirl.create`. I say most common, because there are some places that call `FactoryGirl.create`. Only some mind...